base/deprecated.jl

```diff
@@ -443,3 +443,16 @@ export float32_isvalid, float64_isvalid
 @deprecate (&)(x::Char, y::Char)  Char(UInt32(x) & UInt32(y))
 @deprecate (|)(x::Char, y::Char)  Char(UInt32(x) | UInt32(y))
 @deprecate ($)(x::Char, y::Char)  Char(UInt32(x) $ UInt32(y))
+
+# 11241
+
+@deprecate is_valid_char(ch::Char) isvalid(ch)
+@deprecate is_valid_char(ch::Union(Unsigned, Integer)) isvalid(Char, ch)
```

Comment:
That's what I had, originally, but you had said to change to using the single argument form...
and since I've tried to make any changes that the reviewers have requested, I did so...
Do you now want me to change them back to the way I had them before?